tests/sklearn/test_sklearn_autolog.py

```diff
@@ -206,37 +204,52 @@ def test_classifier():
     # use RandomForestClassifier that has method [predict_proba], so that we can test
     # logging of (1) log_loss and (2) roc_auc_score.
     model = sklearn.ensemble.RandomForestClassifier(max_depth=2, random_state=0, n_estimators=10)
-    X, y_true = get_iris()
+
+    # use binary datasets to cover the test for roc curve & precision recall curve
+    X, y_true = sklearn.datasets.load_breast_cancer(return_X_y=True)
 
     with mlflow.start_run() as run:
         model = fit_model(model, X, y_true, "fit")
 
     y_pred = model.predict(X)
     y_pred_prob = model.predict_proba(X)
+    # For bonary classification, y_score only accepts the probability of greater label
+    y_pred_prob_roc = [prob[1] for prob in y_pred_prob]
```

Comment:
```suggestion
    # For bonary classification, y_score only accepts the probability of greater label
    y_pred_prob_roc = y_pred_prob[:, 1]
```

Reply:
What is the span stuff?

Reply:
`[:, 1]` means `grab the second column`. Does this answer your question?

Reply:
Oh, not sure why but what I saw was a bunch of <span> stuff (looks like html). Now it looks normal again.
Thanks!